README.rst

```diff
@@ -0,0 +1,4 @@
+Python Docs Sphinx Theme
+=========================
+
+This is the theme for the Python documentation
```

Comment:
The note from the python-dev discussion can just be added here as a second paragraph:

> Note that when adopting this theme, you're also borrowing an element of the trust and credibility established by the CPython core developers over the years. That's fine, and you're welcome to do so for other Python community projects if you so choose, but please keep in mind that in doing so you're also choosing to accept some of the responsibility for maintaining that collective trust.